Muehle_Logic.ipynb

```diff
@@ -307,14 +291,14 @@
     "    currentPlayerMills = set()\n",
     "\n",
     "    #find all current mills on the board\n",
-    "    for x in range (0,3):\n",
-    "        for i in [0,2,4,6]:\n",
-    "                if(pieces[1][x][i] == pieces[1][x][i+1] == pieces[1][x][(i+2) % 8] != 0):\n",
-    "                    new_mills.add((pieces[1][x][i], x ,i))\n",
+    "    for ring in range (0,3):\n",
+    "        for cell in [0,2,4,6]:\n",
+    "                if(pieces[1][ring][cell] == pieces[1][ring][cell+1] == pieces[1][ring][(cell+2) % 8] != 0):\n",
+    "                    new_mills.add((pieces[1][ring][cell], ring ,cell))\n",
     "\n",
-    "    for i in [1,3,5,7]:\n",
-    "        if(pieces[1][0][i] == pieces[1][1][i] == pieces[1][2][i] != 0):\n",
-    "            new_mills.add((pieces[1][x][i], 0 ,i))\n",
```

Comment:
das x steht in diesem Fall nicht zwingend für Ring. Da hier die vertikalen Mühlen erkannt werden würde ich ein 'v'/'vertikal' eintragen, dann kann schon nichts verwechselt werden.

Reply:
EGAL mach ich ja

Reply:
https://github.com/luisafuss/muehle-studienarbeit/issues/49